sdk/core/Azure.Core.TestFramework/src/PlaybackTransport.cs

```diff
@@ -121,6 +121,12 @@ public Response GetResponse(RecordEntry recordEntry)
                 {
                     response.AddHeader(new HttpHeader(responseHeader.Key, value));
                 }
+
+                // Ignore the Retry-After header wait time for 429 responses during playback.
+                if (response.Status == 429 && response.Headers.TryGetValue("Retry-After", out _))
```

Comment:
This won't work when we switch to an out-of-process test proxy. We might need to file a feature request for them.

Reply:
cc @JoshLove-msft